composer.json

```diff
@@ -19,14 +19,16 @@
         "composer/installers": "^2.1",
         "cweagans/composer-patches": "^1.7",
         "drupal/admin_toolbar": "^3.3",
+        "drupal/config_split": "^2.0@RC",
         "drupal/core-composer-scaffold": "^10.0",
         "drupal/core-recommended": "^10.0",
         "drupal/monolog": "^3.0@beta",
         "drupal/simplei": "^2.0@beta",
         "drush/drush": "^11.4",
         "vlucas/phpdotenv": "^5.4",
         "webflo/drupal-finder": "^1.2",
-        "wunderio/drupal-ping": "^2.4"
+        "wunderio/drupal-ping": "^2.4",
+        "wunderio/updates_log": "^2"
```

Comment:
I noticed there's this profile web/profiles/custom/basic/basic.info.yml Is it supposed to be used during fresh Drupal projects? Should we enable the updates_log there? It had one bug that it could not find CKEditor atm.
![2023-10-04_13-31](https://github.com/wunderio/drupal-project/assets/492375/13bb222d-cc49-4b9e-bc74-45605c384ca3)
![2023-10-04_13-30](https://github.com/wunderio/drupal-project/assets/492375/2ebda2ac-c4fe-43c8-b1b7-07af1e1d6cdc)

@tormi

Reply:
This has been on hold for a while. My position is that we should remove the `basic` profile and switch to `minimal` with config in `config/sync` folder. Then we can export config changes more easily. Also, we can implement `config_split` in a way it's been done in our projects.